### PR TITLE
update to feathers buzzard and angular 5.2.0

### PR DESCRIPTION
**Pull request by j2L4e**
also added legacy 'feathers' module typings stub

### CODE REVIEW

File: src/app/services/feathers.service.ts [hunk 1]
```diff
@@ -1,34 +1,33 @@
 import { Injectable } from '@angular/core';
 
-import * as feathers from 'feathers/client';
 import * as feathersRx from 'feathers-reactive';
 import * as io from 'socket.io-client';
-import * as hooks from 'feathers-hooks';
-import * as socketio from 'feathers-socketio/client';
-import * as authentication from 'feathers-authentication-client';
 
+import feathers from '@feathersjs/feathers';
```
Comment (daffl) [inline src/app/services/feathers.service.ts:6]: Does the ng CLI transpiler include `node_modules`? If it works it probably does (or just works on newer browsers).

Comment (j2L4e) [reply src/app/services/feathers.service.ts:6]: The TypeScript default target is ES5, so `import feathers from '@feathersjs/feathers'` transpiles to `const feathers = require('@feathersjs/feathers').default`. TS default imports are different from babel's.
The angular CLI just webpacks node_modules, there's no further transpilation.

Comment (daffl) [reply src/app/services/feathers.service.ts:6]: Sorry, I meant will the code in `@feathersjs/feathers` be transpiled as well? Some module loaders include `node_modules` by default (assuming that it is already ES5 which is not the case here).

Comment (daffl) [reply src/app/services/feathers.service.ts:6]: Feathers core is using ES6 syntax but not ES module imports.

Comment (j2L4e) [reply src/app/services/feathers.service.ts:6]: I checked a standard prod build and everything feathers related is still es6 (I only looked for arrow functions, though).

**Pull request merged**